packages/tracer/src/types/Tracer.ts

```diff
@@ -29,21 +29,70 @@ type TracerOptions = {
 /**
  * Options for handler decorators and middleware.
  *
- * Usage:
+ * Options supported:
+ * * `captureResponse` - (_optional_) - Disable response serialization as subsegment metadata
+ * 
+ * Middleware usage:
+ * @example
+ * ```typescript
+ * import middy from '@middy/core';
+ * 
+ * const tracer = new Tracer();
+ * 
+ * const lambdaHandler = async (_event: any, _context: any): Promise<void> => {};
+ * 
+ * export const handler = middy(lambdaHandler)
+ *  .use(captureLambdaHandler(tracer, { captureResponse: false }));
+ * ```
+ * 
+ * Decorator usage:
  * @example
  * ```typescript
  * const tracer = new Tracer();
  *
  * class Lambda implements LambdaInterface {
  *   @tracer.captureLambdaHandler({ captureResponse: false })
- *   async handler(_event: any, _context: any): Promise<void> {}
+ *   public async handler(_event: any, _context: any): Promise<void> {}
+ * }
+ *
+ * const handlerClass = new Lambda();
+ * export const handler = handlerClass.handler.bind(handlerClass);
+ * ```
+ */
+type CaptureLambdaHandlerOptions = {
+  captureResponse?: boolean
+};
+
+/**
+ * Options for method decorators.
+ * 
+ * Options supported:
+ * * `subSegmentName` - (_optional_) - Set a custom name for the subsegment
+ * * `captureResponse` - (_optional_) - Disable response serialization as subsegment metadata
+ *
+ * Usage:
+ * @example
+ * ```typescript
+ * const tracer = new Tracer();
+ *
+ * class Lambda implements LambdaInterface {
+ *   @tracer.captureMethod({ subSegmentName: 'gettingChargeId', captureResponse: false })
+ *   private getChargeId(): string {
+ *     return 'foo bar';
+ *   }
+ *
+ *   @tracer.captureLambdaHandler({ subSegmentName: '', captureResponse: false })
```

Comment:
Should `subSegmentName` be in this `@tracer.captureLambdaHandler()` example?

Reply:
```suggestion
 *   @tracer.captureLambdaHandler({ captureResponse: false })
```